pkg/chunk/aws/storage_client.go

```diff
@@ -248,6 +248,12 @@ func (a storageClient) BatchWrite(ctx context.Context, input chunk.WriteBatch) e
 				unprocessed.TakeReqs(requests, -1)
 				backoff.Wait()
 				continue
+			}else if ok && awsErr.Code() == "ValidationException"{
```

Comment:
Can we get the desired effect without a hard-coded string?
Next-best would be a string constant.

Reply:
Had to go with next best thing. The dynamo library doesn't have it's own exported constant for this error, like they do for `dynamodb.ErrCodeProvisionedThroughputExceededException`